storage-transfer/system-test/transfer.test.js

```diff
@@ -16,49 +16,46 @@
 /* eslint no-empty: 0 */
 'use strict';
 
-const {Storage} = require(`@google-cloud/storage`);
+const {Storage} = require('@google-cloud/storage');
 const storage = new Storage();
-const test = require(`ava`);
-const tools = require(`@google-cloud/nodejs-repo-tools`);
-const uuid = require(`uuid`);
+const assert = require('assert');
+const tools = require('@google-cloud/nodejs-repo-tools');
+const uuid = require('uuid');
 
-const program = require(`../transfer`);
+const program = require('../transfer');
 
 const firstBucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
 const secondBucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
 
 let jobName;
-const date = `2222/08/11`;
-const time = `15:30`;
-const description = `this is a test`;
-const status = `DISABLED`;
+const date = '2222/08/11';
+const time = '15:30';
+const description = 'this is a test';
+const status = 'DISABLED';
 
-test.before(tools.checkCredentials);
-test.before(async () => {
+before(async () => {
+  tools.checkCredentials();
   tools.stubConsole();
 
   const bucketOptions = {
     entity: 'allUsers',
     role: storage.acl.WRITER_ROLE,
   };
-  await storage.createBucket(firstBucketName).then(data => {
-    const bucket = data[0];
-    return bucket.acl.add(bucketOptions);
-  });
-  await storage.createBucket(secondBucketName).then(data => {
-    const bucket = data[0];
-    return bucket.acl.add(bucketOptions);
-  });
+  const [bucket1] = await storage.createBucket(firstBucketName);
+  await bucket1.acl.add(bucketOptions);
+  const [bucket2] = await storage.createBucket(secondBucketName);
+  await bucket2.acl.add(bucketOptions);
 });
 
-test.after.always(async () => {
+after(async () => {
   tools.restoreConsole();
   const bucketOne = storage.bucket(firstBucketName);
   const bucketTwo = storage.bucket(secondBucketName);
   try {
```

Comment:
Just for readability, could this be:

```js
// Attempt to do the delete twice, and swallow any errors
for (let i=0; i<2; i++) {
  await bucketOne.deleteFiles({force: true}).catch(console.warn);
}
```

Reply:
Don't you dare adding `awaits`, that breaks everything. Trust me, we've been there.